### PR TITLE
Add unit tests to Response method, readme, and convert to Standard JS

### PR DESCRIPTION
**Pull request by jveldboom-rsc**
This adds the following:

- 100% unit test coverage for `index.js`. I did not add tests for the `Error` models as I'm planning on removing or reworking those in the near future to fit the standards of the backend team.
- Initial README.md to document how to use the library
- Convert code to use [Standard JS](https://standardjs.com/) formatting

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,55 +1,58 @@
...
\ No newline at end of file
+  Response,
+  Errors: {
+    Unauthorized: require('./module/errors/Unauthorized'),
+    UnprocessableEntity: require('./module/errors/UnprocessableEntity'),
+    InternalServerError: require('./module/errors/InternalServerError'),
+    Forbidden: require('./module/errors/Forbidden'),
+    NotFound: require('./module/errors/NotFound'),
+    Fail: require('./module/errors/Fail'),
+    AlreadyExists: require('./module/errors/AlreadyExists')
```
Comment (rsc-rfrench) [inline index.js:56]: This pattern is super interesting! Python's style guide explicitly disallows `import`ing anywhere other than at the top of a file.  But I think this is a novel way to present resources to a caller without having to pretend like I have any interest other than just forwarding those resources.

**Review by rsc-rfrench — APPROVED** [review]
Strong to medium-strong.

File: index.js (hunk 1 above)
Comment (ghost) [reply index.js:56]: I agree @rsc-rfrench , I love this pattern. I've used it for express routing too so I could have my route documentation live in a single file, but have the actual implementation of the route live in another.

**Comment by ghost** [comment]
@jveldboom-rsc was there anything you wanted me to look over in particular :) ?

**Comment by rsc-rfrench** [comment]
@kpalko-rsc I need you to look over that empty profile pic: https://github.com/settings/profile

**Comment by jveldboom-rsc** [comment]
@kpalko-rsc first, nice profile picture. Seond, I just wanted to make you aware of the change and if you saw any issues. Going to push a quick fix to fix some wording in the README, but then merge it in.

Thanks guys!

**Pull request merged**